### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by blackjack4494** [pxb1988/dex2jar#14]
https://drive.google.com/open?id=0ByNGAVfceOvSbnJvbGNNazJEX3M&authuser=0

I guess it is the same error like in #10 #11 #12 #13 
But i don't know if you need more "errors" to bring out a fix.

**Comment by pxb1988** [comment]
hi guy, 

your dex is protected under some protector.

http://www.strazzere.com/papers/DexEducation-PracticingSafeDex.pdf
https://www.defcon.org/images/defcon-22/dc-22-presentations/Strazzere-Sawyer/DEFCON-22-Strazzere-and-Sawyer-Android-Hacker-Protection-Level-UPDATED.pdf

Please get the real dex on runtime and then try it with dex2jar.

**Comment by blackjack4494** [comment]
Yes the protector being used is ProGuard. I just could not efford the time to have a look into it with IDA Pro.
I can't get the apk without ProGuard since it is not my application.
I just could see how you can upgrade your app protection with those pdfs you wrote.

What do you mean exactly by getting the real dex on runtime?

**Comment by pxb1988** [comment]
as i known, proguard is a just an obfuscator, rename ids, optimize code. it does not encrypt code.

the dex is modified by a protector, the code is encrypted, and decrypt on runtime. so try get the decrypted dex (the real dex) on runtime
